### ISSUE TITLE
Error undefined

### ISSUE THREAD

**Issue by kahwooi** [bojand/grpc-caller#2]
When I hit error at client.sayHello() I got undefined err:
```
try {
 const res = await client.sayHello({ name: 'Bob' })
} catch (err) {
  throw err
}
```
Anyway of getting what is the error I am hitting?

**Comment by bojand** [comment]
Can you please provide more context and information. You can see sample usage here:

https://github.com/bojand/grpc-caller/blob/master/test/basics.test.js#L88

**Comment by kahwooi** [comment]
Yes I am doing just that, but there is an error with the call, and it catches err, but the err is undefined.

I am not able to know what error I am hitting.
